coverity.bat

```diff
@@ -0,0 +1,74 @@
+@ECHO OFF
+
+SETLOCAL
+
+PUSHD %~dp0
+
+IF NOT DEFINED COVDIR SET "COVDIR=C:\cov-analysis"
```

Comment:
@whoisj: we could change this to something you might have set up on your machine.